common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -669,7 +669,8 @@ def convert(content, input_format, output_format):
 
             if output_format == 'txt':
                 text = json.loads(content)['text']
-                return HTMLParser().unescape("\n".join(text))
+                text_without_none = [line if line else 'None' for line in text]
```

Comment:
you can avoid additional loop like `'\n'.join(filter(None, text))`

Reply:
@awaisdar001 this will remove the blank line which is not required in this case

Reply:
Lets confirm the expected behaviour from product then.